src/main/java/com/hubspot/jinjava/lib/filter/DefaultFilter.java

```diff
@@ -65,6 +67,30 @@ else if (object != null) {
     return args[0];
   }
 
+  @Override
+  public Object filter(Object object, JinjavaInterpreter interpreter, Object... args) {
```

Comment:
Couldn't you change all the `String` varags to `Object` varargs instead of adding new methods to every filter?